app/build.gradle

```diff
@@ -30,6 +30,7 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
```

Comment:
remove this file from commit

Reply:
not resolved yet

Reply:
Why are you making this comment resolved? It's not resolved.
This file should not be a part of commit.